File: 00. Estructura/01. Lenguajes de proposito general/02. Programación básica/09. Conversión/MathLab/Conversión.m

% Entero
entero = entero                    	% Entero a Entero
decimal = double(entero)            % Entero a Decimal
caracter = char(entero) 			% Entero a Caracter
booleano = logical(entero)       	% Entero a Booleano
cadena = num2str(entero)		    % Entero a String

% Decimal
entero = int64(decimal)             % Decimal a Entero
decimal = decimal                   % Decimal a Decimal
caracter = char(decimal) 			% Decimal a Caracter
booleano = logical(decimal)       	% Decimal a Booleano
cadena = num2str(decimal)           % Decimal a String

% Caracter
entero = int64(caracter)        			% Caracter a Entero
decimal = double(caracter)					% Caracter a Decimal
caracter = caracter                 		% Caracter a Caracter
booleano = logical(caracter)    			% Caracter a Booleano
cadena = convertCharsToStrings(caracter)    % Caracter a String

% Booleano
entero = int64(booleano)            					% Booleano a Entero
decimal = double(booleano)          					% Booleano a Decimal
caracter = char(int64(booelano))    					% Booleano a Caracter
booleano = booelano                 					% Booleano a Booleano
C = {'false','true'}; cadena = cell2mat(C(1+booelano)); % Booleano a String

% String
entero = int64(str2double(cadena))	% String a Entero
decimal = str2double(cadena)        % String a Decimal
S = char(cadena); caracter = S(1)   % String a Caracter
booleano = cadena ~= ""             % String a Booleano
cadena = cadena                     % String a String

File: 01. Lenguajes de proposito general/02. Programación básica/09. Conversión/Python/Conversión.py

# Entero
entero = entero                    	# Entero a Entero
decimal = float(entero)             # Entero a Decimal
caracter = chr(entero) 				# Entero a Caracter
booleano = bool(entero)       		# Entero a Booleano
cadena = str(entero)		        # Entero a String

# Decimal
entero = int(decimal)               # Decimal a Entero
decimal = decimal                   # Decimal a Decimal
caracter = chr(int(decimal)) 		# Decimal a Caracter
booleano = bool(decimal)       		# Decimal a Booleano
cadena = str(entero)            	# Decimal a String

# Caracter
entero = ord(caracter)        		# Caracter a Entero
decimal = float(ord(caracter))		# Caracter a Decimal
caracter = caracter                 # Caracter a Caracter
booleano = (ord(caracter) != 0)     # Caracter a Booleano
cadena = caracter                   # Caracter a String

# Booleano
entero = int(booleano)              # Booleano a Entero
decimal = float(booleano)           # Booleano a Decimal
caracter = chr(booelano)            # Booleano a Caracter
booleano = booelano                 # Booleano a Booleano
cadena = str(booleano)        		# Booleano a String

# String
entero = int(cadena)               	# String a Entero
decimal = float(cadena)            	# String a Decimal
caracter = cadena[0]            	# String a Caracter
booleano = bool(cadena)             # String a Booleano
cadena = cadena                     # String a String

